modules/libcom/src/osi/compiler/gcc/compilerSpecific.h

```diff
@@ -60,6 +60,9 @@
 /*
  * No return marker
  */
+#ifndef vxWorks
+// VxWorks does not mark abort() or exit() noreturn!
```

Comment:
With #606 merged, we can allow single line comments in C code now?

Reply:
You mean C++ comments?

Reply:
The MSVC-2010 compiler was the last one complaining about those. I just double-checked the VxWorks 6.9 compiler, it has no problem with "C++-style" or "single-line" `//` comments in a .c file.

Reply:
I will change it to a C style comment.

Reply:
Why? That's no longer necessary, we have dropped support for those old compilers now.

Reply:
I don't actually like to find C++ style comments in C code. But I did not notice immediately because no compiler complains any more. Even Codacy is fine with that. Alas, where have the good old times gone?

Reply:
> You mean C++ comments?

Technically these have been standard since C99.


> ... Alas, where have the good old times gone?

`gcc -Werror=kids-these-days ...` ;)